integration_tests/src/main/python/map_test.py

```diff
@@ -96,27 +96,20 @@ def test_map_expr_multiple_pairs():
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : gen_df(spark, data_gen).selectExpr(
                 'map("key1", b, "key2", a) as m1',
-                'map(a, b, b, a) as m2'),
-                conf={'spark.rapids.sql.createMap.enabled':True})
+                'map(a, b, b, a) as m2'))
 
```

Comment:
nit: Can we add a very small test with duplicate guaranteed keys so we can verify that both are throwing exceptions properly?